src/memefactory/ui/my_settings/page.cljs

```diff
@@ -23,14 +24,17 @@
 
 (defn my-settings []
   (let [public-key-sub (re-frame/subscribe [::gql/query {:queries [[:config [[:ui [:public-key]]]]]}])
-        active-account (re-frame/subscribe [::accounts-subs/active-account])
         form-data (r/atom {})
         errors (reaction {:local (cond-> {}
                                    (not (valid-email? (or (:email @form-data) "")))
                                    (assoc-in [:email :error] "Invalid email format"))})]
     (fn []
       (let [public-key (-> @public-key-sub :config :ui :public-key)
+            active-account (re-frame/subscribe [::accounts-subs/active-account])
             settings (re-frame/subscribe [:memefactory.ui.subs/settings @active-account])]
+        (when @active-account
```

Comment:
This is not really correct approach, since the dispatch could be called many times, on every react render. The right approach would be to use https://github.com/district0x/district-ui-router#watch-active-page-1 to listen when settings page was visited. 

To correctly start listening at app bootstrap, you should make it as mount module, example of such code can be seen in memefactory.ui.config https://github.com/district0x/memefactory/blob/53b0dd6d851849c4b5543b4ff4f4d3a3724daf98/src/memefactory/ui/config.cljs#L89-L90. So memefactory.ui.my-settings.events can be such module as well. 

The problem you might encounter is after `watch-active-page` fires your event, the active-account might to be loaded yet. If that's the case, you need to set up one-time listener, that'll wait until active-account is loaded and then performs email load. You'd set up the listener using https://github.com/Day8/re-frame-async-flow-fx, for example: 
```
{:async-flow {:first-dispatch [::do-nothing*]
                        :rules [{:when :seen?
                                   :events [:district.ui.web3-accounts.events/active-account-changed]
                                   :halt? true
                                   :dispatch [::load-email-settings]}]}}
```